sentry/src/db/campaign.rs

```diff
@@ -50,11 +48,90 @@ pub async fn fetch_campaign(
     Ok(row.as_ref().map(Campaign::from))
 }
 
+pub async fn get_campaigns_by_channel(
```

Comment:
We have to see how we will paginate the results. Postgres could block if there are many campaigns for a single channel.

We might need to use a `LIMIT` query multiple times to fetch all campaigns.
Please add a TODO and a Task in the issue #409